src/app/todos/page.js

```diff
@@ -36,7 +36,12 @@ export default function ToDos() {
     }
 
     function removeTodo({ index }) {
-        setTodos(todos.filter((v,idx) => idx!==index));
+        const todoToDelete = todos[index];
+        fetch(`/api/todos/${todoToDelete.id}`, { method: "DELETE" }).then((response) => {
```

Comment:
this doesn't hit your new delete method, but rather the one in the other route file under todos/[id]/route.js

Reply:
I realized that the DELETE route was already written after I wrote the other DELETE route